[NCCL] Enable send/recv tests

mingzhe09088 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#45994 [NCCL] Enable send/recv tests**

Send/Recv tests were disabled because of the https://github.com/pytorch/pytorch/issues/42517. With that issue fixed, this diff enables those tests.

Differential Revision: [D24172484](https://our.internmc.facebook.com/intern/diff/D24172484/)

Commit:
[NCCL] Enable send/recv tests

Send/Recv tests were disabled because of the https://github.com/pytorch/pytorch/issues/42517. With that issue fixed, this diff enables those tests.

Differential Revision: [D24172484](https://our.internmc.facebook.com/intern/diff/D24172484/)

[ghstack-poisoned]

Commit:
Update on "[NCCL] Enable send/recv tests"


Send/Recv tests were disabled because of the https://github.com/pytorch/pytorch/issues/42517. With that issue fixed, this diff enables those tests.

Differential Revision: [D24172484](https://our.internmc.facebook.com/intern/diff/D24172484/)

[ghstack-poisoned]

Commit:
Update on "[NCCL] Enable send/recv tests"


Send/Recv tests were disabled because of the https://github.com/pytorch/pytorch/issues/42517. With that issue fixed, this diff enables those tests.

Differential Revision: [D24172484](https://our.internmc.facebook.com/intern/diff/D24172484/)

[ghstack-poisoned]

Commit:
Update on "[NCCL] Enable send/recv tests"


Send/Recv tests were disabled because of the https://github.com/pytorch/pytorch/issues/42517. With that issue fixed, this diff enables those tests.

Differential Revision: [D24172484](https://our.internmc.facebook.com/intern/diff/D24172484/)

[ghstack-poisoned]

jiayisuse commented:
seems jenkins is failing https://ci.pytorch.org/jenkins/job/pytorch-builds/job/pytorch-linux-bionic-rocm3.8-py3.6-test1/451/console

Commit:
Update on "[NCCL] Enable send/recv tests"


Send/Recv tests were disabled because of the https://github.com/pytorch/pytorch/issues/42517. With that issue fixed, this diff enables those tests.

Differential Revision: [D24172484](https://our.internmc.facebook.com/intern/diff/D24172484/)

[ghstack-poisoned]

mingzhe09088 commented:
> seems jenkins is failing https://ci.pytorch.org/jenkins/job/pytorch-builds/job/pytorch-linux-bionic-rocm3.8-py3.6-test1/451/console

Yes. That's a new test laned yesterday. I've rebased this diff and modified that test.
